editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -4172,7 +4172,7 @@ void SpatialEditor::set_state(const Dictionary &p_state) {
 		Array vp = d["viewports"];
 		ERR_FAIL_COND(vp.size() > 4);
 
-		for (uint32_t i = 0; i < VIEWPORTS_COUNT; i++) {
```

Comment:
```
editor/plugins/spatial_editor_plugin.cpp: In member function 'void SpatialEditor::set_state(const Dictionary&)':
editor/plugins/spatial_editor_plugin.cpp:4175:26: error: comparison of integer expressions of different signedness: 'uint32_t' {aka 'unsigned int'} and 'int' [-Werror=sign-compare]
   for (uint32_t i = 0; i < vp.size(); i++) {
                        ~~^~~~~~~~~~~
```
BTW might be worth changing the hardcoded `4` on the line above to `VIEWPORTS_COUNT`.

Reply:
I thought about that as well hehehe, I just wanted to keep the diff short.
I'll change it.